### PR TITLE
Make clear that Python is not mandatory

### PR DESCRIPTION
**Pull request by ronaldotd**
Although we are allowing Elixir and Go applications, the project specification still suggests that Python is mandatory.

### CODE REVIEW

**Comment by rougeth** [comment]
Now it's not clear that the application must be written in one of the 3 accepted languages.  Maybe a line in the `Project Requirements` section.

File: README.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ These calls are between the numbers 99988526423 (source) and 9993468278 (destina
 ## Project Requirements:
 
 * Provide a working environment with your project (eg. Heroku, )
+* Application must be written in Python, Elixir or Go.
```
Comment (osantana) [inline README.md]: `* Application mus be written in one of those languages: Python, Go or Elixir.`?

Comment (luizdepra) [reply README.md]: A forma original ta ok para mim.

**Comment by rougeth** [comment]
Vou deixar o merge pro @osantana que comentou também.

File: README.md (hunk 1 above)
Comment (ronaldotd) [reply README.md]: A ordem das linguagens indicaria alguma preferência @osantana?

Comment (osantana) [reply README.md]: > A ordem das linguagens indicaria alguma preferência @osantana?

não deveria...

**Pull request merged**